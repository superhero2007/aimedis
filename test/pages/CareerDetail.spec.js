import React from 'react';
import { shallow } from 'enzyme';

import CareerDetail from 'pages/CareerDetail';

function setup() {
  return shallow(<CareerDetail />);
}

describe('CareerDetail', () => {
  const wrapper = setup();

  it('should be a StatelessComponent', () => {
    expect(wrapper.instance()).toBeNull();
  });

  it('should render properly', () => {
    expect(wrapper.html()).toMatchSnapshot();
  });
});
