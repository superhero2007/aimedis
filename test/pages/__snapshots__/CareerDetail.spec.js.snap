// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`CareerDetail should render properly 1`] = `"<div class=\\"app__career__detail app__route\\"><p class=\\"d-none\\">1</p><div class=\\"container\\"><section class=\\"app__career__detail__section1\\"><div class=\\"app__career__detail__section1__content\\"><div class=\\"row\\"><div class=\\"col-12 col-lg-7\\"><p class=\\"app__career__detail__section1__content__title\\">Web Developer</p><p class=\\"app__career__detail__section1__content__subtitle m-0\\">React native / JS, Node JS, NoSQL, PHP</p><div class=\\"app__career__detail__section1__content__info\\"><div class=\\"app__career__detail__section1__content__info__element\\"><div class=\\"app__career__detail__section1__content__info__element__icon\\"><i class=\\"i-map-marker\\"></i></div><div class=\\"app__career__detail__section1__content__info__element__text\\">VENLO</div></div><div class=\\"app__career__detail__section1__content__info__element\\"><div class=\\"app__career__detail__section1__content__info__element__icon\\"><i class=\\"i-clock-o\\"></i></div><div class=\\"app__career__detail__section1__content__info__element__text\\">FULL-TIME</div></div></div><p class=\\"app__career__detail__section1__content__description\\">At World Vision, we&#x27;re sharing a big dream for children. One that moves beyond just clean water or food or healthcare to offer a full solution to poverty.</p><p class=\\"app__career__detail__section1__content__description\\">We want children to have everything they need to thrive, both physically and spiritually, so they can experience the kind of life God intends life in all its fullness.</p><p class=\\"app__career__detail__section1__content__subtitle\\">Required Experience :</p><div class=\\"app__career__detail__section1__content__description\\"><ul><li>Bachelor’s degree in design, digital art, interaction design or related field or equivalent work experience (eight years).</li><li>Previous work experience implementing, managing and / or leading within the digital creative field, 8-10 years, preferably in an agency (internal or external) environment.</li><li>At least 5 years of UX design as a key, hands-on interaction / visual designer of a UI / UX team through the product development cycle.</li><li>Expansive knowledge of layout, typography, mobile technology and web-based graphics and animations, and expert knowledge of design programs including Adobe Creative Suite, Sketch, InVision.</li></ul></div><p class=\\"app__career__detail__section1__content__description\\">Motion design skills preferred. A portfolio of sample projects will be requested.</p><p class=\\"app__career__detail__section1__content__subtitle\\">Benefits :</p><div class=\\"app__career__detail__section1__content__description\\"><ul><li>Bachelor’s degree in design, digital art, interaction design or related field or equivalent work experience (eight years).</li><li>Previous work experience implementing, managing and / or leading within the digital creative field, 8-10 years, preferably in an agency (internal or external) environment.</li><li>At least 5 years of UX design as a key, hands-on interaction / visual designer of a UI / UX team through the product development cycle.</li><li>Expansive knowledge of layout, typography, mobile technology and web-based graphics and animations, and expert knowledge of design programs including Adobe Creative Suite, Sketch, InVision.</li></ul></div><p class=\\"app__career__detail__section1__content__reference mb-1\\">In order to apply to this posotion, please send:</p><div class=\\"app__career__detail__section1__content__description m-0\\"><ul><li>CV</li><li>Cover Letter</li><li>link to LinkedIn profile (if there is one)</li></ul></div><p class=\\"app__career__detail__section1__content__reference mt-1\\">to email address <span>jobs@aimedis.com</span></p></div></div></div></section><section class=\\"app__career__detail__mask d-none d-lg-block\\"><div class=\\"app__career__detail__mask__one\\"><img src=\\"\\" alt=\\"No Asset\\"/></div><div class=\\"app__career__detail__mask__two\\"><img src=\\"\\" alt=\\"No Asset\\"/></div></section></div></div>"`;
